doc/classes/InputEventJoypadButton.xml

```diff
@@ -49,13 +50,13 @@
 	</methods>
 	<members>
 		<member name="button_index" type="int" setter="set_button_index" getter="get_button_index">
-			Joypad button identifier, one of the JOY_BUTTON_* constants in [@Global Scope].
+			Button's identifier, one of the JOY_BUTTON_* constants in [@Global Scope].
 		</member>
 		<member name="pressed" type="bool" setter="set_pressed" getter="is_pressed">
-			Pressed state of the joypad button.
+			[code]true[/code] if the button is pressed. [code]false[/code] if the button is released.
 		</member>
 		<member name="pressure" type="float" setter="set_pressure" getter="get_pressure">
-			Intensity of the button pressure, ranges from 0 to 1.0.
+			Button's pressure intensity, ranges from 0 to 1.0.
```

Comment:
`Represents the pressure the user puts on the button with his finger, if the controller supports it. Ranges from [code]0[/code] to [code]1[/code].`